tools/pepsi/src/upload.rs

```diff
@@ -38,6 +38,40 @@ pub struct Arguments {
     /// The NAOs to upload to e.g. 20w or 10.1.24.22
     #[arg(required = true)]
     pub naos: Vec<NaoAddress>,
+    #[arg(long)]
+    pub skip_check_os: bool,
+}
+
+pub async fn check_os(nao: &Nao) -> bool {
+    let os_stable = "5.1.1";
+    let output = nao
+        .ssh_to_nao()
+        .arg("cat /etc/os-release")
+        .output()
+        .await
+        .unwrap();
+
+    let request = String::from_utf8(output.stdout).unwrap();
+    let lines = request.lines();
+    for line in lines {
+        if line.contains("VERSION_ID") {
+            let line_split: Vec<&str> = line.split('=').collect();
+            let os_version = line_split[1];
+
+            println!("Unstable version on {}", nao.host);
+            println!("Use '--skip-check-os' if you still want to proceed");
+            print!("Installed os: {}, ", os_version);
+            println!("Stable os: {}", os_stable);
```

Comment:
- Please correctly capitalize OS.
- You can put the variable into the format string.
- Write single line prints in one line. It better to see the line string in one source code line IMHO.

```suggestion
            println!("Installed OS: {os_version}, stable OS: {os_stable}");
```

Reply:
Ok